salt/lax/config/etc-nginx-sitesenabled-bot-lax.conf

```diff
@@ -0,0 +1,66 @@
+# the upstream component nginx needs to connect to
+upstream flask {
+    server unix:///tmp/bot-lax-uwsgi.sock;
+}
+
+{% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+# close connection on those that don't specify a 'host' header
+# http://nginx.org/en/docs/http/request_processing.html
+server {
+    listen 80;
+    listen 443;
+    server_name "";
+    return 444;
+}
+{% endif %}
+
+# configuration of the server
+server {
+    # we always listen on port 8001
+    # in non-dev environments port 8001 is only available to internal traffic
+    {% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+    listen      8001 ssl; # see /etc/nginx/nginx.conf for global ssl settings
```

Comment:
probably needs to be opened in `projects/elife.yaml`, at least to internal traffic (plus `bldr update_template` on all lax instances)